workshops/views.py

```diff
@@ -1061,16 +1062,19 @@ def event_import(request):
 
         if 'slug' not in tags:
             # there are no HTML tags, so let's try the old method
-            url = generate_url_to_event_index(url)
+            index_url = generate_url_to_event_index(url)
 
             # fetch page
-            response = requests.get(url)
+            response = requests.get(index_url)
 
             if response.status_code == 200:
                 # don't throw errors for pages we fall back to
                 content = response.text
                 tags = find_tags_on_event_index(content)
 
+                if 'slug' not in tags:
+                    tags['slug'] = urlparse(url).path.replace('/', '')
```

Comment:
This assumes more about the URL than I'd like.  Can we do something like:

    Event.WEBSITE_REGEX.match(url).group('repo')

?  We can leave it to `generate_url_to_event_index` to notice and complain about URLs that don't match `WEBSITE_REGEX`.